cfme/generic_objects/definition/ui.py

```diff
@@ -17,6 +17,7 @@
 from cfme.utils.appliance.implementations.ui import ViaUI
 
 
+# -------------------- collection methods
 @MiqImplementationContext.external_for(GenericObjectDefinitionCollection.create, ViaUI)
```

Comment:
**Suggestion:**
I think comment is not necessary here as This `GenericObjectDefinitionCollection.create` shows it's an collection method.